app/models/compute_resource.rb

```diff
@@ -262,6 +262,15 @@ def set_console_password=(setpw)
     self.attrs[:setpw] = nil
   end
 
+  # this method is overwritten for Libvirt
```

Comment:
to make setting `display_type` behave in a same way as setting `set_console_password` (successfully do nothing when you try to update the attribute on unsupported resource)